src/main/resources/GoogleApps/GoogleAppsConfigSheet.vm

```diff
@@ -13,82 +13,95 @@ $xwiki.ssx.use('GoogleApps.GoogleAppsConfigClass')
     $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}")##
 #end
 #macro(tp $n $p)##
-    $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p)##
+    #set($prf="GoogleApps.GoogleAppsConfigClass_")
+    #if($n.startsWith("googleapps"))#set($prf="")#end
+    #set($msg=$services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p))##
```

Comment:
There should be a space before and after the `=` sign. 
`##` are more a hack-ish way of hiding something wrong. Could you please try to get rid of them and if it's a display issue, to use a correct solution? 
Same remark for `$n` and `$p`. We highly recommend to use self-explanatory names for variables. As they may be `name` and `parameters`, you can quickly get lost in a block of code containing the entire alphabet, so it's even harder to follow.

Reply:
It appeard indeed that the trailing `## ` did not bother after being removed.
I also reformatted with spaces around the equal signs (all including those from old times).